pyopenms_viz/_core.py

```
@@ -658,35 +729,48 @@ def _kind(self):
+        else:
+            self.data.sort_values(by=self.x, inplace=True)
+        
+        # Convert to relative intensity if required.
+        if self.relative_intensity:
+            self.data[self.y] = self.data[self.y] / self.data[self.y].max() * 100
 
+        # Validate tiling configuration in the constructor
+        if hasattr(self._config, "tile_by"):
+            tile_by = self._config.tile_by
+            if tile_by not in self.data.columns:
+                warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)